CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+## Unreleased
+
+#### Breaking Changes
+
+- `map.featuresAt` and `map.featuresIn` are removed. Use `map.queryRenderedFeatures` or `map.querySourceFeatures` instead. To migrate:
+    - replace `featuresAt` and `featuresIn` with `queryRenderedFeatures`
+    - `queryRenderedFeatures` is synchronous. Remove the callback and use the return value. An async version `queryRenderedFeaturesAsync` exists for really slow queries.
+    - rename the `layer` parameters to `layers` and make it an array of strings.
+    - remove the `radius` paramter. `queryRenderedFeatures` automatically uses style property values when doing queries.
```

Comment:
Ah! That makes sense. This line would've made more sense to me initially if it were written like

 > - remove the `radius` parameter. `queryRenderedFeatures` considers style properties (like `line-width`) with an effective radius of 0 when doing queries.